web/package.json

```diff
@@ -20,6 +20,7 @@
     "@mui/material": "^5.15.14",
     "@mui/x-date-pickers-pro": "^7.0.0",
     "@mui/x-license": "^7.0.0",
+    "@tanstack/react-query": "^5.40.1",
```

Comment:
🆘 we already have zustand. I don't think we should use two different React state management libs... 🤔